qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -162,17 +162,46 @@ class WebEngineSearch(browsertab.AbstractSearch):
 
     Attributes:
         _flags: The QWebEnginePage.FindFlags of the last search.
+        _flag_wrap: An additional flag indicating whether the last search
+                    used wrapping.
         _pending_searches: How many searches have been started but not called
                            back yet.
+        _nowrap_available: Whether the nowrap functionality is available.
+        _active_match: The 1-based index of the currently active matc
+                       on the page.
+        _total_matches: The total number of search matches on the page.
     """
 
     def __init__(self, tab, parent=None):
         super().__init__(tab, parent)
         self._flags = QWebEnginePage.FindFlags(0)  # type: ignore
+        self._flag_wrap = True
         self._pending_searches = 0
+        # The API necessary to stop wrapping was added in this version
+        if qtutils.version_check("5.14"):
+            self._active_match = 0
+            self._total_matches = 0
+            self._nowrap_available = True
+            self._store_match_data_connected = False
+        else:
+            self._nowrap_available = False
+
+    def _store_match_data(self, result):
+        self._active_match = result.activeMatch()
+        self._total_matches = result.numberOfMatches()
+        log.webview.debug("active match: number {0} of {1}"
```

Comment:
```suggestion
        log.webview.debug("Active search match: {}/{}"
```

(to have some more context when reading the log message, and the indices aren't required since Python3)